### DISCUSSION TITLE
Remove space between the query builder items and the results in dialogs

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5899] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
When building and running queries, there is a large gap between the items in the query and the results. This gap can make it difficult to see your query and the items being returned by it at the same time. It is especially problematic in dialogs where there is usually only a small amount of space.

It would be nice if the query items had a maximum height of the entire dialog before they become scrollable but the query results should be displayed right below. This would make it less jarring when interacting with queries in this way.

## Current behavior
https://github.com/specify/specify7/assets/37256050/40c10203-c7b1-48e7-9efc-87429a590cd5

## Desired behavior
<img width="731" alt="image" src="https://github.com/specify/specify7/assets/37256050/3e944a56-c184-4460-9b6d-249808542c4b">

**Comment by maxpatiiuk** [comment]
hmm, I though the gap wasn't supposed to be there when query is embedded.
thus this is more of a bug than an enhancement
